### PR TITLE
Improvements to message formatting

### PR DESCRIPTION
**Pull request by subliun**

### CODE REVIEW

File: communication/message_formatting.md [hunk 1]
```diff
@@ -12,10 +12,4 @@ Message Formatting
 - ![](/badge/req.png) A sending client must not insert or remove any bytes from
   the original input.
 
-- ![](/badge/req.png) For each line of a received message, if the line begins
-  with the `>` character, the client must render that line in a different
-  color.
-    -Green is the recommended colour, but a client may choose whichever colour
-    it wants
```
Comment (zetok) [inline communication/message_formatting.md:19]: what should be required, is an opt-out option for marking quoting green

**Comment by stal888** [comment]
Quit the memes

**Comment by JFreegman** [comment]
> No description provided.

The whole reason we made this public before we finished it was to incite discussion about how to improve it.

**Comment by Zer0-One** [comment]
Is greentext really an interoperability concern? Maybe that was a mistake...

**Comment by subliun** [comment]
@JFreegman 

> ASCII removed

Because a client that doesn't work for the majority of the world should not be "endorsed/supported by the Tox Project"

> Greentext requirement removed

Because obviousness

**Comment by JFreegman** [comment]
I agree that clients should only support UTF-8. I'm pretty indifferent about greentext.

**Comment by optimumtact** [comment]
utf-8 only makes sense, it's 2015, ascii is dead.

green text is required.

**Comment by stal888** [comment]
@optimumtact actually, greentext is only a recommendation. clients can use which ever colour they want.

(in fact, i would change it to "visually distinct" instead of another colour)

**Comment by subliun** [comment]
@stal888 I would accept that.

**Comment by installgen2** [comment]
Marking quotes is essential functionality, and "greentext" is a simple and widely understood way of doing so.

**Comment by JFreegman** [comment]
Changes have been merged in two separate commits: https://github.com/Tox/Tox-Client-Standard/commit/b54a5dfd9ee3a50acede8ed3034dbbd485209257 https://github.com/Tox/Tox-Client-Standard/commit/bc59bbec8b18fc0102724a00e95c15f565ca6c2e

**Pull request closed without merging**